src/hooks/web3/index.tsx

```diff
@@ -43,7 +43,12 @@ export interface ProviderProps {
 }
 
 export function TestableProvider({ provider, children }: PropsWithChildren<{ provider: JsonRpcProvider }>) {
-  const connectors = useMemo(() => [initializeWeb3ReactConnector(JsonRpcConnector, { provider })], [provider])
+  const connector = useMemo(() => initializeWeb3ReactConnector(JsonRpcConnector, { provider }), [provider])
+  useEffect(() => {
+    connector[0].activate()
+  }, [connector])
+
```

Comment:
the changes in this file are copied from #368 . same question here - are they intentional and how do they relate to the test?

Reply:
See https://github.com/Uniswap/widgets/pull/368/files#r1067308234